resume/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Section from "./components/Section";
+import resumeJSON from "./data/resume";
+import Basics from "./components/sections/Basics";
+import Work from "./components/sections/Work";
+
+import "./App.css";
+
+function App() {
+  let counter = 0;
+  const sections = Object.keys(resumeJSON).map(section => {
+    counter++;
+    return <Section title={section} info={resumeJSON[section]} key={counter} />;
+  });
```

Comment:
Just FYI that could be relevant: https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order